CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 1.9.5 - 2019-02-12
+This is a maintenance release.
+
+Maintenance:
+  * Update comments to reflect that we only support JRE 8 in some of our products.
+  * Use `luseradd` instead of `useradd` to fix systemd-tmpfiles-setup failure during reboot due to ldap dependencies of the puppet userid/groups.
```

Comment:
Did this have a ticket number associated with it? I don't see it in the commit history, but I feel like I remember seeing a ticket on our board?

Reply:
Whoops yes it did!

Reply:
Updated with ticket #